templates/elements/quote/quote.html.json

```diff
@@ -1,7 +1,8 @@
 {
-  "name": "Jane Doe",
-  "bio": "Jane Doe's title<br/>Jane Doe's company",
-  "text": "This is the actual quote. Lorem ipsur adipisicing elit, sed do eiusmod tempor incididunt ut labore et...",
-  "image": "../../../img/headshot.jpg",
-  "altText": "Jane Doe"
+  "name": "John Doe",
+  "detail1": "Detail 1",
```

Comment:
@josephgknox There are no detail variables in this template. What are they used for?
Also, bio is now missing.